eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-nugetfeed.yml

```diff
@@ -109,13 +109,7 @@ jobs:
               nuGetFeedType: external
               publishFeedCredentials: Nuget.org
 
-      - task: AzureFileCopy@2
-        displayName: 'Copy Signed Files to Blob'
-        inputs:
-          SourcePath: '${{ parameters.ArtifactsPath }}'
-          azureSubscription: '${{ parameters.PartnerDropsSubscription }}'
```

Comment:
Please remove these parameters from the pipeline template as well and removing the secrets from the pipeline. We should also please update the docs if they mention these at all.